logrotate.c

```diff
@@ -2317,6 +2378,7 @@ static int rotateLogSet(const struct logInfo *log, int force)
     int numRotated = 0;
     struct logState **state;
     struct logNames **rotNames;
+    struct resumeConfigs *resCfgs;
```

Comment:
The `resumeConfigs` struct can be allocated on the stack so that we do not need to handle its (de)allocation explicitly.